tokio-trace-core/Cargo.toml

```diff
@@ -7,5 +7,8 @@ authors = ["Eliza Weisman <eliza@buoyant.io>"]
 default = []
 test-support = []
 
+[dependencies]
+lazy_static = "1"
```

Comment:
We should pin to a minor release in order to not unexpectedly have the min rustc version bumped.

Reply:
👍